gymnasium/envs/registration.py

```diff
@@ -600,10 +600,16 @@ def load_plugin_envs(entry_point: str = "gymnasium.envs"):
             fn = plugin.load()
             try:
                 fn()
+                logger.warn(f"The environment registration system on `import gymnasium` is deprecated and will be removed in v1.0 ({plugin.value}), please update code to `import {plugin.value}` separately or use to load the module `gym.make('{plugin.value}:env_name-v0')`")
```

Comment:
I'm pretty sure this will print out a bunch of warnings whenever you import gymnasium, if you happen to have shimmy/minigrid/etc installed, which is not a great thing. See out previous discussion about how to deprecate it, it might not be possible to do in a sane way.

Reply:
I can't remember what we discussed as the alternative solution. I can't think of another way of deprecating this with just putting a warning here for now